frontend/www/js/omegaup/arena/arena.js

```diff
@@ -1494,6 +1499,98 @@ export class Arena {
           self.myRuns.filter_problem(problem.alias);
         }
 
+        function showQualityNominationPopup() {
+          self.qualityPayload = self.currentProblem.quality_payload;
+          if (typeof self.qualityPayload === 'undefined') {
+            // Quality Nomination only works for Courses
+            return;
+          }
+          if (Object.keys(self.qualityNominationForm).length !== 0) {
+            self.qualityNominationForm.nominated =
+              self.qualityPayload.nominated;
+            self.qualityNominationForm.nominatedBeforeAC =
+              self.qualityPayload.nominatedBeforeAC;
+            self.qualityNominationForm.solved = self.qualityPayload.solved;
+            self.qualityNominationForm.tried = self.qualityPayload.tried;
+            self.qualityNominationForm.dismissed =
+              self.qualityPayload.dismissed;
+            self.qualityNominationForm.dismissedBeforeAC =
+              self.qualityPayload.dismissedBeforeAC;
+            self.qualityNominationForm.canNominateProblem =
+              self.qualityPayload.canNominateProblem;
+            self.qualityNominationForm.problemAlias =
+              self.qualityPayload.problemAlias;
+            return;
+          }
+          self.qualityNominationForm = new Vue({
+            el: '#qualitynomination-popup',
+            render: function(createElement) {
+              return createElement('qualitynomination-popup', {
+                props: {
+                  nominated: this.nominated,
+                  nominatedBeforeAC: this.nominatedBeforeAC,
+                  solved: this.solved,
+                  tried: this.tried,
+                  dismissed: this.dismissed,
+                  dismissedBeforeAC: this.dismissedBeforeAC,
+                  canNominateProblem: this.canNominateProblem,
+                  problemAlias: this.problemAlias,
+                },
+                on: {
+                  submit: function(ev) {
+                    let contents = {};
+                    if (!ev.solved && ev.tried) {
+                      contents.before_ac = true;
+                    }
+                    if (ev.difficulty !== '') {
+                      contents.difficulty = Number.parseInt(ev.difficulty, 10);
+                    }
+                    if (ev.tags.length > 0) {
+                      contents.tags = ev.tags;
+                    }
+                    if (ev.quality !== '') {
+                      contents.quality = Number.parseInt(ev.quality, 10);
+                    }
+                    API.QualityNomination.create({
+                      problem_alias: self.qualityPayload.problem_alias,
+                      nomination: 'suggestion',
+                      contents: JSON.stringify(contents),
+                    }).fail(UI.apiError);
+                  },
+                  dismiss: function(ev) {
+                    let contents = {};
```

Comment:
```suggestion
                    const contents = {
                      before_ac: !ev.solved && ev.tried,
                    };
```

igual arriba, con todas las propiedades que puedas darle inline.

Reply:
Listo